airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -224,6 +226,17 @@ def transform_snowflake(config: Dict[str, Any]):
         else:
             dbt_config["password"] = config["password"]
         return dbt_config
+    
+    def get_jdbc_param_value(parameter,jdbc_url_params,default_value):
+        parameter_length=len(parameter) + 1 # +1 because of the = sign
+        parameters=jdbc_url_params.split(',')
+        parameter_value=default_value
+        
+        for i in parameters:
+            if i.startswith(parameter):
+                parameter_value=urllib.parse.unquote(i[parameter_length:])
+                                
+                return parameter_value
```

Comment:
This is going to return only the first parameter.
Also if don't find any parameter won't return the default value.

Please add unit test for this function.

Reply:
Thanks, I will take a look at it and adjust it.